### ISSUE TITLE
Different Behavior than fuzzywuzzy

### ISSUE THREAD

**Issue by mtshrmn** [rapidfuzz/RapidFuzz#32]
Just as the title says - fuzzywuzzy and rapidfuzz output different results.
Here's an example:

```py
# rapidfuzz:
from rapidfuzz import process
process.extractOne("A", ["B", "C", "D"]) # -> None

# fuzzywuzzy:
from fuzzywuzzy import process
process.extractOne("A", ["B", "C", "D"]) # -> ("B", 0)
```

The line responsible for that is [this](https://github.com/maxbachmann/rapidfuzz/blob/cceb7cb2eaee799d545f34c79fbad56d55711dbd/src/rapidfuzz/process.py#L174)

If you intend to keep this, please note that it's not backward compatible, please document this behavior.

**Comment by mtshrmn** [comment]
The exact commit that introduced this divergence: https://github.com/maxbachmann/rapidfuzz/commit/a4bfbeb2f52e2687fb70a188d146becd9d980320

**Comment by maxbachmann** [comment]
Thanks for bringing this up. This is fixed in 0.9.1 and I started to add some unit tests to prevent this error in future releases.
